managed/utils/encryption/encryption.go

```diff
@@ -68,6 +66,55 @@ func New(keyPath string) *Encryption {
 	return e
 }
 
+// RotateEncryptionKey is a wrapper around DefaultEncryption.RotateEncryptionKey.
+func RotateEncryptionKey() error {
+	err := backupOldEncryptionKey()
+	if err != nil {
+		return err
+	}
+
+	defaultEncryptionMtx.Lock()
+	DefaultEncryption = New()
+	defaultEncryptionMtx.Unlock()
+
+	return nil
+}
+
+// RotateEncryptionKey is will backup old encryption key and generate new one.
+func (e *Encryption) RotateEncryptionKey() error {
+	err := e.backupOldEncryptionKey()
+	if err != nil {
+		return err
+	}
+
+	enc := New()
+	e.Key = enc.Key
+	e.Path = enc.Path
+	e.Primitive = enc.Primitive
+
+	return nil
+}
+
+// RestoreOldEncryptionKey is a wrapper around DefaultEncryption.RestoreOldEncryptionKey.
+func RestoreOldEncryptionKey() error {
+	err := os.Rename(fmt.Sprintf("%s_old.key", strings.TrimSuffix(encryptionKeyPath(), ".key")), encryptionKeyPath())
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
restore should be in the same file as backup. As far as I see, backup is internal function, not public